hoot-core/src/main/cpp/hoot/core/io/ApiDbReader.cpp

```diff
@@ -605,21 +593,24 @@ void ApiDbReader::read(const OsmMapPtr& map)
   }
   else
   {
-    Envelope bounds;
-    if (!_overrideBounds.isNull())
+    std::shared_ptr<geos::geom::Geometry> bounds;
+    if (_overrideBounds.get())
     {
       bounds = _overrideBounds;
     }
     else
     {
       bounds = _bounds;
     }
+    // api db bounds reading is based off of quad tiles which only operates on rectangular bounds,
```

Comment:
This is one a few examples where the bounds must be treated as an `Envelope`. Have noted the others as well.